README.md

```diff
@@ -153,76 +153,135 @@ On certain pages an API endpoint is called to receive experiment analysis data f
+WARNING: this will remove your database and all data. Use this to reset your dev environment.
+
+#### make migrate
 
-Start a dev server listening on port 80 using the [Django runserver](https://docs.djangoproject.com/en/1.10/ref/django-admin/#runserver)
+Apply all django migrations to the database.  This must be run after removing database volumes before starting a dev instance.
 
-### up_db
+#### make load_locales_countries
+
+Populates locales and countries in the database from the [Firefox Product Details package](https://pypi.org/project/django-mozilla-product-details/)
+
+#### load_dummy_experiments
+
+Populates the database with dummy experiments of all types/statuses using the test factories
+
+#### make refresh
+
+Run kill, migrate, load_locales_countries load_dummy_experiments.  Useful for resetting your dev environment when switching branches or after package updates.
+
+### Running a dev instance
+#### make up
+
+Start a dev server listening on port 80 using the [Django runserver](https://docs.djangoproject.com/en/1.10/ref/django-admin/#runserver).  It is useful to run `make refresh` first to ensure your database is up to date with the latest migrations and test data.
+
+#### make up_db
 
 Start postgresql, redis, autograph, kinto on their respective ports to allow running the Django runserver and yarn watchers locally (non containerized)
 
-### up_django
+#### up_django
 
 Start Django runserver, Celery worker, postgresql, redis, autograph, kinto on their respective ports to allow running the yarn watchers locally (non containerized)
 
-### up_detached
+#### up_detached
 
-Start all containers in the background (not attached to shell)
+Start all containers in the background (not attached to shell).  They can be stopped using `make kill`.
 
-### check
+### Running tests and checks
 
-Run all test and lint suites, this is run in CI on all PRs and deploys
+#### make check
 
-### py_test
+Run all test and lint suites, this is run in CI on all PRs and deploys.
 
-Run python tests
+#### make py_test
 
-### migrate
+Run only the python test suite.
 
-Apply all django migrations
+#### make bash
 
-### load_locales_countries
+Start a bash shell inside the container.  This lets you interact with the containerized filesystem and run Django management commands.
 
-Populates locales and countries
+##### Helpful Python Tips
+You can run the entire python test suite without coverage using the Django test runner:
 
-### load_dummy_experiments
+        ./manage.py test
 
-Populates db with dummy experiments
+For faster performance you can run all tests in parallel:
 
-### bash
+        ./manage.py test --parallel
 
-Start a bash shell inside the container (this lets you interact with the containerized filesystem and run Django management commands)
+You can run only the tests in a certain module by specifying its Python import path:
 
-### ssl
+        ./manage.py test experimenter.experiments.tests.api.v5.test_serializers
```

Comment:
Feel free to not do this, but I constantly use the tip you told me about failing a test so you can snag its import path from the terminal output. Could make a helpful tip.